### DISCUSSION TITLE
Zoom with Scroll Wheel doesn't work

### DISCUSSION THREAD

**Discussion by Lefraudeur** [d2phap/ImageGlass#1584] [Q&A]
I can't zoom with scroll wheel on ImageGlass_Moon_9.0.4.416_Beta-4_x64_net7
I added this to the config file :
```
"MouseWheelActions": {
    "Scroll": "Zoom"
  }
```
following the example here
https://github.com/d2phap/ImageGlass/releases/tag/9.0.2.904

**Comment by d2phap** [comment]
That's the correct config, but by default, IG already uses mouse wheel scrolling for zooming.
Maybe you should attach a video?

**Reply by Lefraudeur** [reply]
Nevermind, I tested on another clean computer, and it works well, it seems to be an issue on my side. Maybe I've done some weird things with my windows. It's strange because, as you can see in the video, the scroll wheel is detected since I can browse images with it.
https://github.com/d2phap/ImageGlass/assets/91006387/26674637-3ae9-4513-8821-438e32cf16f1
